src/Modal/Modal.js

```diff
@@ -1,26 +1,31 @@
 import React, {Component} from "react";
-import './Modal.css'
+import './Modal.scss'
 import {ReactComponent as Heart} from "../assets/heart.svg";
 
 
 export default class Modal extends Component {
     state = {
         isOpen: false
     }
+
+    stateTrue = () => {
+        this.setState({ isOpen: true});
+    }
+    stateFalse = () => {
+        this.setState({ isOpen: false});
+    }
+
     render() {
```

Comment:
stateTrue = () => {
        this.setState({ isOpen: true});
    }
    stateFalse = () => {
        this.setState({ isOpen: false});
    }
    make this as one fucntion